backend/public/app/src/behaviors/csync-behavior.html

```diff
@@ -199,8 +199,13 @@
           if (!value.exists) {
             return
           }
-          this.currentViewerIds[value.creator] = Date.now()
-          this.set("currentViewerCount", Object.keys(this.currentViewerIds).length)
+          let currentTimeInSecondsFrom2001 = ((Date.now() - this.timeOffsetMilliseconds) / 1000).toFixed(6)
+
+          // Filtering out invalid heartbeats that are older than one second.
```

Comment:
Invalid heartbeats are older than 10 seconds on iOS. I think we should keep that consistant